src-built-in/components/appCatalog2/src/app.jsx

```diff
@@ -284,12 +290,25 @@ export default class AppMarket extends React.Component {
 
 		return (
 			<div>
-				<SearchBar hidden={page === "showcase" ? true : false} backButton={page !== "home"} tags={tags} activeTags={activeTags} tagSelected={this.addTag}
-					removeTag={this.removeTag} goHome={this.goHome} installationActionTaken={this.state.installationActionTaken}
-					search={this.changeSearch} isViewingApp={this.state.activeApp !== null} />
-				<div className="market_content">
-					{pageContents}
-				</div>
+				{this.state.apps.length === 0 && !this.state.isLoading &&
+					<div className='server-error'>
+						<br />
+						<br />
+						<span>There are no apps loaded to the catalog.</span>
```

Comment:
I understand that this is the exact wording from the mockup, however I do not want this language in the system.

Possible alternatives:

> No applications were provided for the catalog. Please consult your system admin.

> The catalog did not receive any applications. Please consult your system admin.

> The catalog has no applications. Please consult your system admin.

There are a number of ways to reword this so the language doesn't sound broken.